cw-relayer/tests/e2e/server/price_server.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"net"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	oracletypes "github.com/ojo-network/ojo/x/oracle/types"
+	"google.golang.org/grpc"
+)
+
+type Server struct {
+	oracletypes.UnimplementedQueryServer
+	mockPrices []types.DecCoin
+}
+
+func (s *Server) ExchangeRates(context.Context, *oracletypes.QueryExchangeRates) (*oracletypes.QueryExchangeRatesResponse, error) {
+	return &oracletypes.QueryExchangeRatesResponse{
+		ExchangeRates: s.mockPrices,
+	}, nil
+}
+
+func (s *Server) setMockPrices() {
+	for i := 0; i < 5; i++ {
+		price := rand.Float64()
+		if i%2 == 1 {
+			// to have prices above 1
+			price = price * 100000
+		}
+
+		priceDec := types.MustNewDecFromStr(strconv.FormatFloat(price, 'f', 9, 64))
+		s.mockPrices = append(s.mockPrices, types.NewDecCoinFromDec(fmt.Sprintf("TEST-%v", i), priceDec))
```

Comment:
There's really no sdk.Dec randomizer baked into the SDK? We might want to put this into https://github.com/ojo-network/ojo/tree/main/util/decmath if we reuse it

Reply:
Looks much better with the randomizer btw 🔥

Reply:
Not sure if there is one in the sdk